apps/src/lib/kits/maker/CircuitPlaygroundBoard.js

```diff
@@ -204,6 +204,13 @@ export default class CircuitPlaygroundBoard extends EventEmitter {
     });
   }
 
+  reset() {
+    const {led, buzzer, colorLeds} = this.prewiredComponents_;
+    led.off();
+    colorLeds.forEach(led => led.off());
+    buzzer.off();
```

Comment:
Any other state we need to reset here?

Reply:
I'm a little concerned that this question didn't get an answer.  There's almost certainly board state you've missed in this list.  The one that comes to mind is control of whether we're streaming certain sensor data, which can have an impact on board performance.  I believe this also causes colorLed state to persist across runs - you've called `off()` but if the first command on next run is `on()` I think it will preserve the color from the last run.

Was this approach compared to using the [Firmata "system reset" command `0xFF`](https://github.com/firmata/protocol/blob/master/protocol.md) and reinitializing without closing and reopening the serial connection.